rnr_control/src/rnr_control.c

```diff
@@ -216,6 +292,24 @@ static void handle_arguments( ps_node_ref const node_ref, int argc, char **argv
                 }
             }
         }
+        else if( optret == 'f' )
```

Comment:
This is an ok stop-gap, but the final version of this application should get the SDF based on the logfile session provided by the `-t` flag. 

Once that's implemented we can remove this command line option.